load-test/run_load_test.sh

```diff
@@ -36,7 +36,7 @@ if [ "$loops" == "" ]; then
 fi
 
 # make this var point to your installed CMDRunner location
-command_runner="/usr/local/Cellar/jmeter/3.3/libexec/lib/ext/CMDRunner.jar"
```

Comment:
This is an unnecessary change as it just shows an example of path. It can be very different for users based on local path

Reply:
Oh looks like my working version got tagged on here. Reversed it back to 3.3